src/main/java/org/jenkinsci/plugins/deployment/HostRecords.java

```diff
@@ -0,0 +1,45 @@
+package org.jenkinsci.plugins.deployment;
+
+import hudson.EnvVars;
+import hudson.model.AbstractBuild;
+import hudson.model.EnvironmentContributingAction;
+import hudson.model.InvisibleAction;
+import org.apache.commons.lang.StringUtils;
+
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * @author Felix Belzunce Arcos
+ * @since May 8, 2015
```

Comment:
Since should refer the version (just add TODO). A summary description is commonly more interesting for API users that the author's name ;)